browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -5238,8 +5243,20 @@ L.CanvasTileLayer = L.Layer.extend({
 			rectangles.push(rectQueue);
 		}
 
-		for (var r = 0; r < rectangles.length; ++r)
+		for (var r = 0; r < rectangles.length; ++r) {
+
+			// While we are actively scrolling, filter out duplicate
+			// (still) missing tiles requests during the scroll.
+			if (this._moveInProgress) {
+				var rectStr = JSON.stringify(rectangles[r]);
```

Comment:
Quick skim - but surely we should track the tiles we're asking for and not the rectangles here and ensure we don't fetch a tile 'pixel' from that 'bitmap' twice (?)

Reply:
this looked simple

Reply:
I -think- in the tilecombine logic we already have tile-by-tile de-duplication & aggregation logic that we could re-factor and re-use, but was just a quick skim =)